eventfiles/urls.py

```diff
@@ -0,0 +1,7 @@
+from . import views
+from django.urls import path
+
+urlpatterns = [
+    path('event_file/download/', views.download, name="download"),
+    path("event_file/<str:event_id>", views.file_page_per_event, name="main"),
```

Comment:
Can you change the name from main,
In my opinion, "main" can be misleading because of the home page,
you can change it to upload_files

The same can be said about line 5:
you can change it to download_files

Reply:
Done